tests/test_config.py

```diff
@@ -39,5 +39,4 @@ def test_init_creation_of_buildtest_dir():
 
     shutil.rmtree(BUILDTEST_ROOT)
     init()
```

Comment:
Since we know this is in a user's home, this test needs to run to be based in tmp_path. We can specify the BUILDTEST_ROOT to be there.

Reply:
since ``init`` takes no argument the only way i can do this is set ``BUILDTEST_ROOT`` to ``tmp_path`` by updating variable in ``buildtest.defaults`` module so that init is using tmp_path. Do you think that what you want?

Reply:
ok i was able to tweak the way we do ``init`` and creation process so that regression test can work with ``tmp_path`` see https://github.com/HPC-buildtest/buildtest-framework/pull/277/commits/e1f9598e3d8e49cb12ad2606dc6ba80561165b36 More importantly we can now pass any directory path where to store configuration file. Since 'site' is a sub-directory in BUILDTEST_ROOT, we could even alter where ``buildtest get``   will clone the repo. Generally user may not want stuff cloned in $HOME due to quota limitation.